.ci/openvino-onnx/Dockerfile

```diff
@@ -75,7 +75,7 @@ RUN make -j $(nproc) install
 
 # Run tests via tox
 WORKDIR /openvino/runtime/bindings/python
-ENV ngraph_DIR=/openvino/dist/deployment_tools/ngraph
+ENV OpenVINO_DIR=/openvino/dist/deployment_tools/inference_engine/share
```

Comment:
Is it a temporary change?

Reply:
`OpenVINO_DIR` is permanent, path will be changed in future.